test/src/main/java/org/axonframework/test/matchers/MatchAllFieldFilter.java

```diff
@@ -44,12 +43,17 @@ public MatchAllFieldFilter(Collection<FieldFilter> filters) {
     }
 
     @Override
-    public boolean accept(Field field) {
+    public boolean accept(String field) {
         for (FieldFilter filter : filters) {
             if (!filter.accept(field)) {
                 return false;
             }
         }
         return true;
     }
+
+    // TODO: Broken Liskov Substitution Principle!
+    public List<FieldFilter> getFilters() {
+        return filters;
```

Comment:
Is there a better solution that the one breaking the Liskov Substitution Principle?